### ISSUE TITLE
Is the changelog available anywhere?

### ISSUE THREAD

**Issue by adimangla** [Paytm-Payments/Paytm_Android_App_Kit#25]
Hi, our app uses Android SDK v1.0.1 at the moment. We want to know how far back we are as compared to the current stable/production release and what the major changes since then are. 

We want to know if its worth upgrading or not.

Thanks!

**Comment by AironVarun** [comment]
@Paytm Team : Kindly plan to add change logs in your git repository for other to track what changes have been done.

**Comment by lastw0nd3r** [comment]
Yes @Paytm-Payments-Admin, it will be easy to figure out whether or not to upgrade SDK from 2.0 to 2.1 or to x.x version.

**Comment by ashish-dhiman** [comment]
Yes we should be able to track what changes have been done.
